indexer/config/presets.go

```diff
@@ -0,0 +1,63 @@
+package config
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// in future presets can just be onchain config and fetched on initialization
+
+// Mapping of l2 chain ids to their preset chain configurations
+var presetL1Contracts = map[int]L1Contracts{
+	// OP Mainnet
+	10: {
+		OptimismPortal:         common.HexToAddress("0xbEb5Fc579115071764c7423A4f12eDde41f106Ed"),
+		L2OutputOracle:         common.HexToAddress("0xdfe97868233d1aa22e815a266982f2cf17685a27"),
+		L1CrossDomainMessenger: common.HexToAddress("0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1"),
+		L1StandardBridge:       common.HexToAddress("0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"),
+		L1ERC721Bridge:         common.HexToAddress("0x5a7749f83b81B301cAb5f48EB8516B986DAef23D"),
+	},
+	// OP Goerli
+	420: {
+		OptimismPortal:         common.HexToAddress("0x5b47E1A08Ea6d985D6649300584e6722Ec4B1383"),
+		L2OutputOracle:         common.HexToAddress("0xE6Dfba0953616Bacab0c9A8ecb3a9BBa77FC15c0"),
+		L1CrossDomainMessenger: common.HexToAddress("0x5086d1eEF304eb5284A0f6720f79403b4e9bE294"),
+		L1StandardBridge:       common.HexToAddress("0x636Af16bf2f682dD3109e60102b8E1A089FedAa8"),
+		L1ERC721Bridge:         common.HexToAddress("0x8DD330DdE8D9898d43b4dc840Da27A07dF91b3c9"),
+	},
+	// Base Mainnet
+	8453: {
+		OptimismPortal:         common.HexToAddress("0x49048044D57e1C92A77f79988d21Fa8fAF74E97e"),
+		L2OutputOracle:         common.HexToAddress("0x56315b90c40730925ec5485cf004d835058518A0"),
+		L1CrossDomainMessenger: common.HexToAddress("0x866E82a600A1414e583f7F13623F1aC5d58b0Afa"),
+		L1StandardBridge:       common.HexToAddress("0x3154Cf16ccdb4C6d922629664174b904d80F2C35"),
+		// FIXME update this to the correct address
+		L1ERC721Bridge: common.HexToAddress("0x0000000000000000000000000000000000000000"),
```

Comment:
Gonna have to find these via this other_bridge property. Since we aren't using anything other than OP atm I skipped for now 
![image](https://github.com/ethereum-optimism/optimism/assets/35039927/33125927-1a21-494a-b32d-19bf2251e8ba)

Reply:
Could consider not specifying it and just index it but seems pointless to add that complexity when onchain/onrpc config is coming

Reply:
I'm also open to simply removing this contract address from the list until we get around to supporting the 721 bridge